packages/core/src/extension/extension-request-sender.ts

```diff
@@ -0,0 +1,28 @@
+import { RequestSender, Response } from '@bigcommerce/request-sender';
+
+import {
+    ContentType,
+    INTERNAL_USE_ONLY,
+    RequestOptions,
+    SDK_VERSION_HEADERS,
+} from '../common/http-request';
+
+import { Extension } from './extension';
+
+export const EXTENSIONS_API_URL = '/api/storefront/checkout-extensions';
+
+export class ExtensionRequestSender {
+    constructor(private _requestSender: RequestSender) {}
+
+    loadExtensions({ timeout, params }: RequestOptions = {}): Promise<Response<Extension[]>> {
+        return this._requestSender.get(EXTENSIONS_API_URL, {
+            timeout,
```

Comment:
What does this timeout do?

Reply:
I think we offer `timeout`for almost all requests, it is a standard option in `@bigcommerce/request-sender`.
> Provide this option if you want to cancel or time out the request. If the timeout object completes before the request, the request will be cancelled.

Reply:
Think seems like a copy paste issue, as the client will need to pass `timeout` as options, which we rarely do 🤔